### DISCUSSION TITLE
How to copy an asset file to a new place

### DISCUSSION THREAD

**Discussion by xulihang** [tauri-apps/tauri#5195] [Q&A] [answered]
I need to bundle some installers in my app. The program will prompt the users to save the installers to a folder for the users to run the installers.

But I couldn't find how to do this.

**Comment by xulihang** [comment, marked as answer]
I ended up moving the files to the resources.

**Reply by wcly** [reply]
set `resources` in `tauri.conf.json`.

example:
```
 "resources": ["tmp/"]
```

dir: 
```
--src-tauri
 --tmp
  --xxx.text
```
